### DISCUSSION TITLE
How do I remove the visibility of previews inside a post?

### DISCUSSION THREAD

**Discussion by SeiOkami** [cotes2020/jekyll-theme-chirpy#1804] [Q&A]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

Some posts need to have a preview in the list. But on the page of the post itself there was no preview. To avoid duplication (as in the image below)

![image](https://github.com/cotes2020/jekyll-theme-chirpy/assets/42138875/9bb00835-4b7b-42a2-b0e2-c22f75029ed1)

### Operations you have already tried

I managed to make my own modifications to the file, but I want to do without it

_layouts/post.html

```
      {% if page.image %}

<<<
        {% if page.image.in_post == true or page.image.in_post == nil %}
>>>
        {% capture src %}src="{{ page.image.path | default: page.image }}"{% endcapture %}
        {% capture class %}class="preview-img{% if page.image.no_bg %}{{ ' no-bg' }}{% endif %}"{% endcapture %}
        {% capture alt %}alt="{{ page.image.alt | xml_escape | default: "Preview Image" }}"{% endcapture %}

        {% if page.image.lqip %}
          {%- capture lqip -%}lqip="{{ page.image.lqip }}"{%- endcapture -%}
        {% endif %}

        <div class="mt-3 mb-3">
          <img {{ src }} {{ class }} {{ alt }} w="1200" h="630" {{ lqip }}>
          {%- if page.image.alt -%}
            <figcaption class="text-center pt-2 pb-2">{{ page.image.alt }}</figcaption>
          {%- endif -%}
        </div>

<<<
        {% endif %}
>>>

      {% endif %}
```

**Comment by kungfux** [comment]
> I managed to make my own modifications to the file, but I want to do without it

I don't think there is a way to do this. If `image` is defined, it will appear on both the home page and the post page. This is by design. The same goes for the post description. It may be strange to see the content on the home page, but not have it in the post.

In this particular case, you can have an introductory section before the video to have a space between the post cover and the video with the same preview image to avoid visual duplication.

**Comment by walkbori** [comment]
Do you want "show on the list and hide in the post", right?
(but, it is applied all post)

### _layouts/post.html
```
<!--
   <div class="mt-3 mb-3">
          <img {{ src }} {{ class }} {{ alt }} w="1200" h="630" {{ lqip }}>
          {%- if page.image.alt -%}
            <figcaption class="text-center pt-2 pb-2">{{ page.image.alt }}</figcaption>
          {%- endif -%}
    </div>
-->
```

**Comment by mattpopovich** [comment]
I followed the suggestions above to make this into a PR: https://github.com/cotes2020/jekyll-theme-chirpy/pull/2053

Seems like the owner wasn't interested in pulling this change into the baseline. 

However, I'm adding these changes to my website. Here's what it looks like to make the changes to a website generated from [chirpy-starter](https://github.com/cotes2020/chirpy-starter): https://github.com/mattpopovich/mattpopovich.com/commit/f14ed5504da748ff6574830f8d4666d80a55bf59
